public/app/plugins/datasource/testdata/components/SimulationQueryEditor.tsx

```diff
@@ -0,0 +1,83 @@
+import { SelectableValue } from '@grafana/data';
+import { InlineField, InlineFieldRow, InlineSwitch, Input, Label, Select } from '@grafana/ui';
+import React, { FormEvent } from 'react';
+import { EditorProps } from '../QueryEditor';
+import { SimulationQuery } from '../types';
+
+export const SimulationQueryEditor = ({ onChange, query }: EditorProps) => {
+  const simQuery = query.sim ?? ({} as SimulationQuery);
+  const simKey = simQuery.key ?? ({} as typeof simQuery.key);
+  const options = [
+    { label: 'Flight', value: 'flight' },
+    { label: 'Sine', value: 'sine' },
```

Comment:
Do we have a way to share values between backend / frontend for easier maintainability? (i.e. if this changes in backend, no clear pointer that this file needs to change as well to not break functionality)

Reply:
We can make an HTTP request, and will -- but I want to keep that out of this PR since it requires adding additional parameters to the query editor props and should be done separately